src/status_im/contexts/wallet/send/from/view.cljs

```diff
@@ -31,10 +30,11 @@
 (defn view
   []
   (let [accounts (rf/sub [:wallet/accounts-without-watched-accounts])]
+    (rn/use-unmount on-close)
```

Comment:
This approach has a problem while developing.

When the code hot-reloaded the components are unmounted, so this will move us to the root of the app, making really hard to develop.

cc: @OmarBasem @Parveshdhull  @ilmotta 

I think we should instead use a callback maybe provided by the navigation library

Reply:
@OmarBasem Have you tried hot-realoading the code after this change?

Reply:
Thanks @ulisesmac, you are right, this can cause a problem. I will look into it.